CONTRIBUTING.md

```diff
@@ -58,7 +58,7 @@ If you'd like to preview your documentation changes, first commit your changes t
 * `npm install`
 * `npm run docs:view`
 
-Visit `http://localhost:8089` and you should see the docs with your local changes. Make sure you `npm run docs:clean` before committing, because automated generated files to `docs/*` should **not** be in PRs.
+Visit `http://127.0.0.1:8089` and you should see the docs with your local changes. Make sure you `npm run docs:clean` before committing, because automated generated files to `docs/*` should **not** be in PRs.
```

Comment:
This one isn't strictly necessary because Node !== browser. But shouldn't be a problem.

Reply:
i saw this in the mass replace and wanted to keep it just in case a browser also decides to change that meaning and to keep the docs more consistent (or in case it is used as a argument to some node process, like a link checker)